manager/ytsync.go

```diff
@@ -543,6 +563,17 @@ func (s *Sync) doSync() error {
 	if err != nil {
 		return errors.Prefix("error updating remote database", err)
 	}
+	if pubsOnWallet == 0 {
```

Comment:
what is this and why is it here?

Reply:
This is some initial functionality. What it does is check if its been published to already, if not then I want to send the cert to internal-apis. When we deploy this we will start storing those because we need that for the feature. For older channels I need to either to a single run to populate all of them or something else. This is just an initial start and something we will want going forward.